examples/s3/bucket.yaml

```diff
@@ -36,4 +38,7 @@ spec:
           expiration:
             days: 15
   providerConfigRef:
-    name: example
\ No newline at end of file
+    name: example
+  writeConnectionSecretToRef:
+    name: mybucket-1
```

Comment:
What does bucket write to the connection details?

Reply:
Nothing anymore, I think this is a remanent of the previous bucket.yaml for the S3Bucket. Removed now.